lib/debug/debugger.py

```diff
@@ -0,0 +1,192 @@
+# Authors: see git history
+#
+# Copyright (c) 2010 Authors
```

Comment:
Let's set this to 2024.  I have no idea why so many of our files say 2010!  Stating the copyright date doesn't really matter, but seeing 2010 here is weird.

Reply:
done